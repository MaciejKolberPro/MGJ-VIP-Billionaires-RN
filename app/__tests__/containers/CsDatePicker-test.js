import React from 'react'
import renderer from 'react-test-renderer'
import CsDatePicker from '../../containers/CsDatePicker'

test('renders correctly', () => {
  const tree = renderer.create(<CsDatePicker />).toJSON()
  expect(tree).toMatchSnapshot()
})
