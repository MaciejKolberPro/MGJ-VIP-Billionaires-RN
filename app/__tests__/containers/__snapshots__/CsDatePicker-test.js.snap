// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderBottomColor": "grey",
        "borderBottomWidth": 0.5,
        "flexDirection": "row",
        "marginBottom": 10,
      },
      undefined,
    ]
  }
>
  <RNDateTimePicker
    date={1669799718823}
    displayIOS="default"
    enabled={true}
    is24Hour={true}
    mode="date"
    onChange={[Function]}
    onPickerDismiss={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "width": 116,
      }
    }
    testID="date-time-picker"
    textColor="black"
    themeVariant="light"
  />
</View>
`;
